docs/core/windows-prerequisites.md

```diff
@@ -36,7 +36,7 @@ You can see the full set of [supported operating systems](https://github.com/dot
 
 ## .NET Core dependencies
 
-.NET Core requires the VC++ Redistributable when running on Windows. It is installed for you by the .NET Core installer. You need to install the Visual C++ redistributable manually if you are installing .NET Core via the installer script (`dotnet-install.ps1`). 
+.NET Core requires the VC++ Redistributable when running on Windows. It is installed for you by the .NET Core installer. You need to install the Visual C++ redistributable manually if you are installing .NET Core via the installer script (`dotnet-install.ps1`) or are attempting to run a compiled stand alone .NET Core application. 
```

Comment:
[This article](https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/#self-contained-deployments-scd) calls it "self-contained", not "stand alone". I think the terminology should be consistent.

Also, not sure what "compiled" is supposed to mean here, all self-contained apps are compiled.

So, I would change this to:

> … or are attempting to run a self-contained .NET Core application.